libs/features/spectrum2-styles/build.mjs

```diff
@@ -0,0 +1,84 @@
+import fs from 'fs';
+import path from 'path';
+import postcss from 'postcss';
+
+const spectrumCSSPath = path.resolve('node_modules/@spectrum-css/tokens/dist/index.css');
+const miloCSSPath = path.resolve('../../deps/spectrum2-styles.css');
+
+const spectrumCSS = fs.readFileSync(spectrumCSSPath, 'utf8');
+const miloCSS = fs.readFileSync(miloCSSPath, 'utf8');
+
+// Formats and transforms Spectrum CSS Rgb and opacity properties for performance
+const transformRgbProperties = (rule) => {
+  const properties = {};
+
+  const replaceZeroPx = (value) => value.replace(/\b0px\b/g, '0');
+
+  rule.walkDecls((decl) => {
+    const { prop, value } = decl;
+
+    decl.value = replaceZeroPx(value);
+
+    if (prop.endsWith('-rgb')) {
+      const baseName = prop.replace('-rgb', '');
+      properties[baseName] = `rgb(${value.replace(/,\s*/g, ' ')})`;
+      decl.remove();
+    }
+
+    if (prop.endsWith('-opacity')) {
+      const baseName = prop.replace('-opacity', '');
+      properties[baseName] = properties[baseName] ? properties[baseName].replace(')', ` / ${parseFloat(value) * 100}%)`) : `rgb(0 0 0 / ${value})`;
+      decl.remove();
+    }
+  });
+
+  Object.keys(properties).forEach((prop) => rule.append({ prop, value: properties[prop] }));
+};
+
+const extractAndTransformCustomProperties = (css, prefix) => {
+  const customProperties = {};
+  postcss.parse(css).walkRules((rule) => {
+    const { selector } = rule;
+    if (!customProperties[selector]) {
+      customProperties[selector] = {};
+    }
+
+    transformRgbProperties(rule);
+
+    rule.walkDecls((decl) => {
+      let propName = decl.prop;
+      if (propName.startsWith(prefix)) propName = `--spectrum${propName.slice(prefix.length)}`;
+      customProperties[selector][propName] = decl.value;
+    });
+  });
+  return customProperties;
+};
+
+const spectrumCustomProperties = extractAndTransformCustomProperties(spectrumCSS, '--spectrum');
```

Comment:
The names are a little too long and make it hard to read and understand IMO.
```suggestion
const spectrumProps = transformProperties(spectrumCSS, '--spectrum')
```

Reply:
sounds good to me!

Reply:
Sure! I think its important to note this function is actually where the extracting/filtering properties is happening along with the transforming/formatting, if you think transform suffices and is still clear that works for me.

Reply:
sure! I think its important to note this function is extracting as well as formatting/transforming the properties, but I think `transform` should be able to suffice if you think this is still clear to you.